README.md

```diff
@@ -307,6 +308,33 @@ When sending many messages, it's likely that the client needs to send some messa
 
 Make sure your application can survive being blocked for so long.
 
+### SSL
+
+Kafka 0.9+ supports SSL for authentication and authorization. There are two possible modes of operation:
+
+#### You have a 0.9 broker with SSL setup and only want to encrypt traffic to it:
+
+In this case you just need to pass `ssl: true` to `Kafka.new`. You don't need an SSL context.
+
+```ruby
+kafka = Kafka.new(ssl: true, ...)
+```
+
+#### You have a 0.9 broker with SSL setup and want to use client certificates and Kafka's authentication mechanism:
+
+In this case you need to pass `ssl: true`, and an `ssl_context` setup with the client certificate you want to use:
+
+```ruby
+ssl_context = OpenSSL::SSL::SSLContext.new
+ssl_context.set_params(
+  cert: OpenSSL::X509::Certificate.new(YOUR_CLIENT_CERTIFICATE),
+  key: OpenSSL::PKey::RSA.new(YOUR_CLIENT_CERTIFICATE_KEY),
+)
+kafka = Kafka.new(ssl: true, ssl_context: ssl_context ...)
+```
+
+Without these options, ssl support is disabled, and ruby-kafka will throw exceptions when trying to connect to an SSL socket.
```

Comment:
Also: how heterogenous would the configurations be? i.e. would it make sense to simply have options for `ssl_certificate` and `ssl_key`?